test/functional/uiautomator-e2e-specs.js

```diff
@@ -0,0 +1,34 @@
+// transpile :mocha
+
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import 'mochawait';
+import UiAutomator from '../../lib/uiautomator.js';
```

Comment:
should probably import from index to test what the user would be importing---this would have surfaced a bug i believe